src/js/views/globals/modal/modal_views.js

```diff
@@ -21,9 +21,9 @@ const i18n = intl.globals.modal.modalViews;
 const ReplaceElRegion = Region.extend({ replaceElement: true, timeout: 0 });
 
 const SavingFooterView = View.extend({
-  className: 'flex',
+  className: 'flex flex-align-center',
   template: hbs`
-    <div class="button--text" disabled>{{ savingInfoText }}</div>
```

Comment:
`button--text` was adding hover and disabled styling that we don't want here, since this isn't a button.